docs/modules/ROOT/pages/queries.adoc

```diff
@@ -254,3 +270,19 @@ query {
     }
 }
 ----
+
+=== Aggregate undirected relationships
+When performing an aggregation on related nodes, the relationship
+can be defined as an xref::queries.adoc#_undirected_queries[undirected relationship] using the argument `directed: false`:
```

Comment:
**suggestion:** We should be careful about phrasing in relation to this - this makes it sound like the relationship has no direction when in fact we are only querying it disregarding its direction. Perhaps the following?

```suggestion
=== Undirected aggregation queries

When performing an aggregation on related nodes, the query against the relationship
can be defined as an xref::queries.adoc#_undirected_queries[undirected] using the argument `directed: false`:
```